src/types/onyx/Card.ts

```diff
@@ -1,32 +1,17 @@
-type AdditionalData = {
-    isBillingCard?: boolean;
-    isP2PDebitCard?: boolean;
-};
-
-type AccountData = {
-    additionalData?: AdditionalData;
-    addressName?: string;
-    addressState?: string;
-    addressStreet?: string;
-    addressZip?: number;
-    cardMonth?: number;
-
-    /** The masked credit card number */
-    cardNumber?: string;
-
-    cardYear?: number;
-    created?: string;
-    currency?: string;
-    fundID?: number;
-};
+import {ValueOf} from 'type-fest';
+import CONST from '../../CONST';
 
 type Card = {
-    accountData?: AccountData;
-    accountType?: string;
-    description?: string;
-    key?: string;
-    methodID?: number;
-    title?: string;
+    cardID: number;
+    state: number;
+    bank: string;
+    availableSpend: number;
+    domainName: string;
+    maskedPan: string;
+    isVirtual: boolean;
+    fraud: ValueOf<typeof CONST.EXPENSIFY_CARD.FRAUD_TYPES>;
+    cardholderFirstName: string;
+    cardholderLastName: string;
```

Comment:
Would this be optional, or always available?

Reply:
right now always available